docs/run-a-successful-pov.md

```diff
@@ -0,0 +1,172 @@
+---
+slug: run-a-successful-pov
+title: Run a successful POV with Semgrep
+hide_title: true
+description: Explore and deploy the Semgrep platform through a proof-of-value (POV) trial. Semgrep dedicates support and engineering resources to ensure you are able to run Semgrep smoothly.
+tags:
+  - Semgrep AppSec Platform
+  - Support
+---
+
+# Run a successful proof-of-value (POV) with Semgrep
+
+:::note Start a POV
+For any questions or concerns about starting a proof-of-value (POV), reach out to [<i class="fa-regular fa-envelope"></i> sales@semgrep.com](mailto:sales@semgrep.com).
+:::
+
+Run a POV to learn more about Semgrep solutions and receive support that is specific to your infrastructure and business needs. During a POV, you receive dedicated sales, engineering, and support resources to ensure that every Semgrep feature that supports your infrastructure can be implemented quickly and reliably.
+
+## POV requirements
+
+These are some of the requirements the Semgrep team needs from your organization to run a POV. You must decide on these factors.
+
+- **The number of trial licenses**
+    - How many trial licenses are needed? Who among your organization will be evaluating Semgrep?
+- **A method to scan the repositories used in the POV**
+    - **Recommended: Semgrep Managed Scans (SMS)**
+        - This is the fastest way to deploy Semgrep to the repositories you want to scan. It requires code access, which can be limited to only certain repositories. See [Permissions](/deployment/checklist#permissions) to review its requirements.
+    - **CI/CD**
+        - This method relies on a CI configuration file, such as a GitHub Actions workflow file. A CI/CD job must be set for all the repositories you want to scan.
```

Comment:
```suggestion
        - This method relies on a CI configuration file, such as a GitHub Actions workflow file. A CI/CD job must be included in all of the repositories you want to scan.
```

Reply:
I'll go with "created" for this one, i.e. "A CI/CD job must be created for all of the repositories you want to scan."